tests/api/admin/controller/test_custom_lists.py

```diff
@@ -1160,3 +1161,64 @@ def test_auto_update_edit(self, admin_librarian_fixture: AdminLibrarianFixture):
         assert custom_list.auto_update_query == '{"query": "...changed"}'
         assert custom_list.auto_update_status == CustomList.REPOPULATE
         assert [e.work_id for e in custom_list.entries] == [w1.id]
+
+    def test_auto_update_create_unable_to_serialize_query(
+        self,
+        admin_librarian_fixture: AdminLibrarianFixture,
+        db: DatabaseTransactionFixture,
+    ):
+        library = db.default_library()
+        response = admin_librarian_fixture.manager.admin_custom_lists_controller._create_or_update_list(
+            library,
+            "test list",
+            [],
+            [],
+            [],
+            id=None,
+            auto_update=True,
+            auto_update_query={"foo": object()},  # type: ignore[dict-item]
+        )
+
+        assert isinstance(response, ProblemDetail)
+        assert response.status_code == 400
+        assert response.detail == "auto_update_query is not JSON serializable"
+
+    def test_auto_update_create_unable_to_serialize_facets(
+        self,
+        admin_librarian_fixture: AdminLibrarianFixture,
+        db: DatabaseTransactionFixture,
+    ):
+        library = db.default_library()
+        response = admin_librarian_fixture.manager.admin_custom_lists_controller._create_or_update_list(
+            library,
+            "test list",
+            [],
+            [],
+            [],
+            id=None,
+            auto_update=True,
+            auto_update_query={"query": "foo"},
+            auto_update_facets={"foo": object()},  # type: ignore[dict-item]
+        )
+
+        assert isinstance(response, ProblemDetail)
+        assert response.status_code == 400
+        assert response.detail == "auto_update_facets is not JSON serializable"
+
+    def test_auto_update_deleted_entries(
```

Comment:
Do we want a test for entries here, as well? Or maybe there is one somewhere else already?

Reply:
Entries gets some test coverage here:
https://github.com/ThePalaceProject/circulation/blob/965d4225ee09ce95454e72bb0015c6ab8081df37/tests/api/admin/controller/test_custom_lists.py#L368-L395